crates/generated_parser/src/early_error_checker.rs

```diff
@@ -0,0 +1,646 @@
+use crate::context_stack::{
+    BindingInfo, BindingKind, BindingsIndex, BreakOrContinueIndex, ContextMetadata, LabelIndex,
+    LabelInfo, LabelKind,
+};
+use crate::declaration_kind::DeclarationKind;
+use crate::early_errors::*;
+use crate::error::{ParseError, Result};
+use crate::Token;
+use ast::arena;
+use ast::source_atom_set::{SourceAtomSet, SourceAtomSetIndex};
+use std::cell::RefCell;
+use std::rc::Rc;
+
+// Internal Helper Methods for Early Error Checker Labels/Controls
+
+/// Internal Label/Control Helper method:
+/// called From EarlyErrorChecker::check_labelled_statement, and not used by
+/// the struct implementing the trait. This means that
+/// LabelledStatementEarlyErrorsContext is allocated inside this trait.
+fn check_unhandled_continue<'alloc>(
```

Comment:
this method name was misleading.
to me, `unhandled` sounds like there is no corresponding label.
(so, that can be checked only at the script/module/function level)

can you rename this to `check_labelled_continue_to_non_loop` or `check_labelled_continue` that's same as what's done in this function?